puzzlejs/puzzle.js

```diff
@@ -1693,7 +1767,7 @@ function PuzzleGrid(puzzleEntry, options, container, puzzleId) {
     
                 if (edgeCode) { td.setAttribute("data-given-edge-code", edgeCode); }
             }
-            if (cellSavedState) { edgeCode ^= parseInt(cellSavedState[1], 16); }
+            if (cellSavedState && cellSavedState.indexOf(",") != 0) { edgeCode ^= parseInt(cellSavedState[1], 16); }
```

Comment:
Why does this (and the next change) only do work if there's no saved text?

Reply:
This was supposed to be generalized error-proofing to make sure the non-text data existed and was based on what you had already when restoring the spoke levels, but I see now that it's not necessary since there will NEVER be any case where there's saved data that has ONLY text data.  I'll just remove these checks.

Reply:
Sounds good, will revisit if I ever condense the save format like I was obviously considering.